build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: "java-library"
+apply plugin: "java"
+
+
+task copyAllJars(type: Copy) {
```

Comment:
The copy task should be removed.

Reply:
This comment is for the user to copyAllJars from `jpf-core/build` `jpf-symbc/build` to `SPF/build`
For example, if the user wants all Jars to present in the `SPF/build` dir then the user can fire this task manually (Note: this task is not copying the jpf-core jars to jpf-symbc/lib)
If we do not want this task then I can remove it.

Reply:
I have removed this task